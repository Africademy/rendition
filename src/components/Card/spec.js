/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import { Card, Provider } from '../../../dist'

test('Search renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Card>Lorem ipsum dolor so ammet</Card>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
