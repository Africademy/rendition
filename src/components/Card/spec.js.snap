// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
.c4 {
  box-sizing: border-box;
}

.c3 {
  margin-top: 0;
}

.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c2 {
  border: solid 1px #dfdede;
  border-radius: 3px;
  box-shadow: 0 1px 4px 0 rgba(0,0,0,0.14);
  padding: 20px;
  background-color: #ffffff;
  min-height: auto;
}

.c0 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6,
.c0 button {
  font-family: CircularStd,Arial,sans-serif;
}

<div
  className="c0 c1"
>
  <div
    className="c2"
  >
    <div
      className="c3 c4"
    >
      Lorem ipsum dolor so ammet
    </div>
  </div>
</div>
`;
